.pre-commit-config.yaml

```diff
@@ -34,11 +34,6 @@ repos:
     hooks:
       - id: isort
 
-  - repo: https://github.com/asottile/setup-cfg-fmt
-    rev: v1.20.1
-    hooks:
-      - id: setup-cfg-fmt
-
   - repo: https://github.com/mgedmin/check-manifest
```

Comment:
This can probably be dropped too, I think it's (last I checked) not usable with non-setuptools projects (and isn't as important since there's no MANIFEST.in, it uses .gitignore instead)